pandas/core/frame.py

```diff
@@ -4207,17 +4207,23 @@ def update(self, other, join='left', overwrite=True, filter_func=None,
                raise_conflict=False):
         """
         Modify DataFrame in place using non-NA values from passed
-        DataFrame. Aligns on indices
+        DataFrame.
+
+        Aligns on indices.
 
         Parameters
         ----------
         other : DataFrame, or object coercible into a DataFrame
+            Index should be similar to one of the columns in this one. If a
+            Series is passed, its name attribute must be set, and that will be
+            used as the column name in the resulting joined DataFrame.
         join : {'left'}, default 'left'
+            Indicates which column values overwrite.
```

Comment:
Is `join='right'` valid? Can you do some exploration on what happens?

Reply:
It's actually not valid. I tried it.

Reply:
I think you can say "Only left join is implemented, keeping the index and columns of the original object"